src/openapi_parser/specification.py

```diff
@@ -168,8 +168,8 @@ class Parameter:
 class Content:
     type: Union[ContentType, LooseContentType]
     schema: Schema
-    # example: Optional[Any]  # TODO
-    # examples: list[Any] = field(default_factory=list)  # TODO
+    example: Optional[Any]
```

Comment:
```suggestion
    example: Optional[Any] = None
```

This should allow you to omit it from every constructor in the code/tests. `examples` also should be an optional argument for the constructor to keep backward compatibility, please check if you can omit it even without adding `None` as a default value for `example`.

Reply:
yes, agreed, it's quite simple with default value